lighthouse-core/report/html/report-styles.css

```diff
@@ -453,13 +452,20 @@
   margin-right: 0;
 }
 
-.lh-audit__title {
+.lh-audit__display-text {
   flex: 1;
 }
 
+/* Prepend display text with em dash separator. But not in Opportunities. */
+.lh-audit-group:not(.lh-audit-group--load-opportunities) .lh-audit__display-text:not(:empty):before {
```

Comment:
best practices aren't grouped so aren't getting the `:before`

Reply:
fixed